src/react/RegisterDevicePage.js

```diff
@@ -465,14 +105,17 @@ function IntrospectionControlRow({onAddInterface}) {
 
   const [interfaceId, setInterfaceId] = useState(initialState);
```

Comment:
Reading `interfaceId` I expected it to be a string, such as the usual uuidv4. Could `interface` be a better naming?